### PR TITLE
Update fi-FI and ar translations

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Update `fi-FI` translations to be actual Finnish. Fix punction in `ar` translations.

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
Related to: https://github.com/cerner/terra-core/issues/806

File: packages/terra-alert/translations/ar.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "Terra.alert.dismiss": "تجاهل",
```
Comment (emilyrohrbough) [inline packages/terra-alert/translations/ar.json]: Should dismiss also have a `.`?

**Comment by emilyrohrbough** [comment]
I think finish translations were missed for action-header, date-picker, menu, status-view.

**Comment by bjankord** [comment]
@emilyrohrbough 
* [Action Header](https://github.com/cerner/terra-core/blob/master/packages/terra-action-header/translations/fi-FI.json) had Finnish translations already.
* [Menu](https://github.com/cerner/terra-core/blob/master/packages/terra-menu/translations/fi-FI.json) had Finnish translations already.
* [Status View](https://github.com/cerner/terra-core/blob/master/packages/terra-status-view/translations/fi-FI.json) had Finnish translations already.

I'll go update date picker fi-FI translations.

**Comment by bjankord** [comment]
@emilyrohrbough Fixed up en and fi-FI translations in date-picker. I added the fi-FI ones to en by accident. Corrected in this commit: https://github.com/cerner/terra-core/pull/2077/commits/42195240c0a900183962dcaa72e69cdec636bb35

**Pull request merged**